src/tsconfig.json

```diff
@@ -0,0 +1,31 @@
+{
+  "compilerOptions": {
+    "strict": false,
+    "strictNullChecks": false,
+    "noUnusedLocals": false,
+    "noUnusedParameters": false,
+    "noImplicitReturns": false,
+    "noImplicitAny": false,
+    "noImplicitThis": false,
+    "noFallthroughCasesInSwitch": false,
+    "declaration": true,
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "incremental": true,
+    "sourceMap": true,
+    "inlineSourceMap": false,
+    "inlineSources": true,
+    "preserveSymlinks": true,
+    "removeComments": true,
+    "jsx": "react",
+    "lib": ["ES2018", "dom"],
+    "target": "ES2018",
+    "allowJs": true,
+    "esModuleInterop": true,
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "importHelpers": false,
+    "outDir": "../lib"
+  },
+  "compileOnSave": false
```

Comment:
I've never seen this before, and it's also not showing up in the TSConfig reference. Can you explain its purpose?